SeQuant/core/expr.cpp

```diff
@@ -6,12 +6,13 @@
 #include <SeQuant/core/algorithm.hpp>
 #include <SeQuant/core/expr.hpp>
 #include <SeQuant/core/logger.hpp>
-#include <SeQuant/core/runtime.hpp>
```

Comment:
revert

*Rationale*
insignificant changes like reordering of `#include` statements without good reason should be avoided. `#include`s should be organized into space-separated groups (so that clang-format can sort alphabetically within each group) and ordered as follows:

- forward declaration headers (`fwd.h`, etc.)
- headers of the current package
- headers of dependent packages, from more specialized to less specialized, one group per package
- standard headers